kuma/javascript/src/active-banner.jsx

```diff
@@ -55,6 +55,9 @@ import CloseIcon from './icons/close.svg';
 import { getLocale, gettext, Interpolated } from './l10n.js';
 import UserProvider from './user-provider.jsx';
 
+export const DEVELOPER_NEEDS_ID = 'developer_needs';
+export const SUBSCRIPTION_ID = 'subscription_banner';
```

Comment:
The reason I *moved* these to up here was because it felt better that way. Now, there's a new utility function which depends on it and it didn't feel right to have the constants somewhere below.